apps/codelab/src/app/components/angular-test-runner/angular-test-runner.component.ts

```diff
@@ -86,9 +86,9 @@ export class SimpleAngularTestRunnerComponent
       this.tests = handleTestMessage(message, this.tests);
 
       if (message.data.type === 'testEnd') {
-        if (this.tests.length && this.tests.every(test => test.pass)) {
-          this.solved.emit();
-        }
+        this.solved.emit(
+          this.tests.length && this.tests.every(test => test.pass)
```

Comment:
this looks like number|boolean to me, curious why the type checker allows it

Reply:
Sure it could be modified in order to be clear for understanding. In this case a result of every function has always boolean type and all condition will be converted to boolean.

Reply:
This is very minor,
I'd just do `this.tests.length`  -> `!!this.tests.length`

Reply:
> I'd just do `this.tests.length` -> `!!this.tests.length`
I prefer `this.tests.length > 0` as better readable.